packages/cli/src/commands/init.ts

```diff
@@ -1,8 +1,10 @@
 import push from './push';
 import init from '../scripts/init';
+import { promptForArgumentsIfNeeded } from '../utils/prompt';
+import { FileSystem } from 'zos-lib';
 
 const name: string = 'init';
-const signature: string = `${name} <project-name> [version]`;
+const signature: string = `${name} [project-name] [version]`;
 const description: string = `initialize your ZeppelinOS project. Provide a <project-name> and optionally an initial [version] name`;
```

Comment:
We should review these descriptions, especially when it does not add any useful information. "Provide a project-name" is already implied by its arguments. We can make a pass at it in a future PR.

Reply:
Addressed in https://github.com/zeppelinos/zos/issues/727